src/codemodder/codemodder.py

```diff
@@ -40,33 +39,32 @@ def apply_codemod_to_file(
         file_context,
     )
     if not codemod.should_transform:
-        return
+        return False
 
-    logger.info("Running codemod %s for %s", name, file_context.file_path)
     output_tree = codemod.transform_module(source_tree)
     # TODO: there has got to be a more efficient way to check this?
```

Comment:
can delete this comment, deep equals is the right way, efficient or not

Reply:
@clavedeluna I can update the comment but my question here is why do we even need to do tree comparison at all? We already know that we modified the file because we record the change. So in that context this seems needlessly inefficient unless there's something I'm missing.